config/crd/bases/metal3.io_baremetalhosts.yaml

```diff
@@ -57,10 +57,14 @@ spec:
         description: BareMetalHost is the Schema for the baremetalhosts API
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            description: 'APIVersion defines the versioned schema of this representation
```

Comment:
Nothing should be reformatting existing stuff in the CRD. This likely indicates a mismatch of tool versions. AIUI all the tools we use should be pinned to a particular version in the repo (by vendoring or installing a specific version in the Makefile).